spec/spec_helper.rb

```diff
@@ -85,11 +87,7 @@
   #   - http://rspec.info/blog/2012/06/rspecs-new-expectation-syntax/
   #   - http://www.teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/
   #   - http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/#zero-monkey-patching-mode
-  # config.disable_monkey_patching!
-
-  # This setting enables warnings. It's recommended, but in some cases may
-  # be too noisy due to issues in dependencies.
-  # config.warnings = true
+  config.disable_monkey_patching!
```

Comment:
Is why we have to do `RSpec.describe` instead of just `describe`?

Reply:
I think so. I personally like the terse version stylistically, but then thought monkey patching is out of fashion.

What do you prefer?

Reply:
I am fine with `RSpec.describe`.